### DISCUSSION TITLE
🚀 Firecamp Desktop App - Now Available for Download!

### DISCUSSION THREAD

**Discussion by Nishchit14** [firecamp-dev/firecamp#167] [Announcements]
Hello Firecamp community! 👋

I am excited to announce that the Firecamp Desktop App is now available for download! We have worked tirelessly to bring you a seamless and powerful experience with the latest **Firecamp v3** app, and I can't wait for you to try it out.

You can read more about the desktop app on this issue #62

**Download Firecamp from here**
[1. MacOS Intel](https://fcamp.co/mac)
[2. MacOS Silicon](https://fcamp.co/mac-silicon)
[3. Linux AppImage](https://fcamp.co/linux-appImage)
[4. Windows](https://fcamp.co/win-x64)

**Why Firecamp Desktop App?**
Firecamp is designed to streamline your workflow and enhance productivity, whether you're a developer, a project manager, or a QA professional. With its intuitive interface and robust features, Firecamp is the perfect tool to accelerate your API development process. The desktop app will eliminate the browser limitations and It's designed to serve the native experience for individual or teams working collaboratively.

**We Love Feedback!**
Your feedback is crucial to us. If you encounter any issues or have suggestions for improvements, please don't hesitate to reach out. We're here to make Firecamp the best it can be, with your help.

> 📝 Note:
> If you're using the old v2 desktop app, first ensure that your collections are accessible on the web at  https://firecamp.dev. If anything is missing to sync up there, try restarting the V2 app and check again. Then, you can uninstall the V2 app and download the new app from the above link. You can read more about the [v3 release from here.](https://github.com/firecamp-dev/firecamp/releases/tag/3.0.0)

Thank you for being a part of our journey. Happy coding!

Best,
~ND

**Comment by atselvan** [comment]
@Nishchit14  The download links are not working :( 

![image](https://github.com/firecamp-dev/firecamp/assets/26083199/e4734306-c646-41d7-82ba-fe4f3e8cc7e0)

**Reply by Nishchit14** [reply]
I tried it multiple times and it's working fine in my case. I also checked the vercel logs but couldn't find anything helpful. @atselvan Can you please try it again, please confirm whether it's working or not? Also which download link you're facing this issue with?

**Reply by atselvan** [reply]
I tried after a while and it worked 😃

**Reply by Nishchit14** [reply]
Great, Thank you for the confirmation :)

**Reply by recodyx** [reply]
you need an account to create a collection...

sorry but this is not a "postman alternative" for me.

too bad.

**Reply by Nishchit14** [reply]
@recodyx sorry to hear that but as of now it's cloud first. our feature roadmap of collaboration, documentation and AI are tightly coupled with cloud services. and that what we can serve the best. 

We have plan to give local support for REST apis only, but the ETA is not available as of now.

**Reply by meblum** [reply]
![image](https://github.com/firecamp-dev/firecamp/assets/33358938/f9b58463-e226-4577-8bed-85cad0c172ea)

**Reply by Nishchit14** [reply]
@meblum Can you please try now? It seems like the vercel side issue, It must be resolved.

**Reply by meblum** [reply]
Link seems to be working now, but why not publish the binaries as a GitHub release?

**Reply by falahati** [reply]
same issue here as well. gateway error

**Reply by Nishchit14** [reply]
@meblum from the next release, I'll publish binaries on GitHub release tags.

**Comment by Xyndra** [comment]
When will windows be available?

**Reply by Nishchit14** [reply]
@Xyndra I can say one week of ETA. I am waiting to get a code signing certificate.

**Reply by Xyndra** [reply]
@Nishchit14  but you can just use a unsigned exe, it will just give a security popup

**Reply by Alia5** [reply]
@Nishchit14 
Could you please provide build instructions then?  
All I get when opening my build is a white screen...

**Reply by Nishchit14** [reply]
@Xyndra Sure, I will wait for the certificate for two days otherwise will move on with the unsigned .exe

**Reply by Nishchit14** [reply]
@Alia5 You need to start the dev server `pnpm dev` or build the app first with `pnpm build` then you can start building the electron app. May I know the the purpose of building app locally?

**Reply by Alia5** [reply]
@Nishchit14 
Well, doing this, I'll always have to have the dev server running.

The purpose is to not wait for you to publish a windows-build.

**Reply by Nishchit14** [reply]
@Xyndra @Alia5 Thank you for your patience,  You can download the [window app from  here](https://fcamp.co/win-x64)

**Comment by Alia5** [comment]
The windows build lacks a title bar.  
Thus, you cannot move, resize, minimize, etc. the window.

**Reply by Nishchit14** [reply]
Sure, you can track the issue from here. https://github.com/firecamp-dev/firecamp/issues/186

**Comment by just-maik** [comment]
Maybe this is controversal. But one of the main things that drove me towards insomnium instead of insomnia, was the need for a login to create collections and save queries. I dont want my calls to be stored somewhere I have to register for. 

A gamechanger for me would be to offer a local mode (localstorage in browser and some local db on desktop).

Other than that great experience so far.

**Reply by philiphenriksen** [reply]
Same here.
The main reason I want to avoid using Postman at all is _because_ of the requirement to create an account.

I've managed to block Postman from updating to a version that nags about creating accounts, but I would like a better alternative in general, since websockets, for example, cannot be tested without creating accounts.
I heard about this as an alternative to Postman, but _requiring_ the user to log in to do basically anything at all is a big turn-off for me.
